src/sif/bicyclecost.cc

```diff
@@ -434,21 +434,17 @@ class BicycleCost : public DynamicCost {
 // is modulated based on surface type and grade factors.
 
 // Constructor
-BicycleCost::BicycleCost(const Costing costing, const Options& options)
-    : DynamicCost(options, TravelMode::kBicycle) {
-  // Grab the costing options based on the specified costing type
-  const CostingOptions& costing_options = options.costing_options(static_cast<int>(costing));
-
+BicycleCost::BicycleCost(const CostingOptions& options) : DynamicCost(options, TravelMode::kBicycle) {
   // Set hierarchy to allow unlimited transitions
   for (auto& h : hierarchy_limits_) {
     h.max_up_transitions = kUnlimitedTransitions;
   }
 
   // Get the base costs
-  get_base_costs(costing_options);
+  get_base_costs(options);
```

Comment:
Why did you choose to rename this?

Reply:
the underlying type changed from Options to CostingOptions

Reply:
oh jeez. i read the diff in reverse.. i suspect i started at the function signature using the IDE and changed just the type, then i went to the source and renamed the variable inside to match not thinking that it would be less change and make more sense to just rename the argument. sorry for the confusion im running on low sleep :smile: